src/Data/Swagger/Internal.hs

```diff
@@ -482,7 +555,7 @@ data Schema = Schema
   , _schemaRequired :: [ParamName]
 
   , _schemaAllOf :: Maybe [Schema]
-  , _schemaProperties :: HashMap Text (Referenced Schema)
+  , _schemaProperties :: Definitions (Referenced Schema)
```

Comment:
`Definitions` is for definitions that are then used with `$ref`.
Properties are hence not `Definitions`, but `InsOrdHashMap Text`.